py-aquapark

stamaksim opened the pull request.

stamaksim committed:
py-aquapark

vsmutok commented on app/main.py:
```diff
@@ -2,24 +2,73 @@
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError
```
add some description
```suggestion
            raise ValueError(....)
```

vsmutok commented on app/main.py:
```diff
@@ -2,24 +2,73 @@
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.name = "_" + name
```
```suggestion
        self.protected_name = "_" + name
```

stamaksim committed:
py-aquapark
raise ValueError was fixed (added
error description)
descriptor name was changed to protected_name

vsmutok reviewed (approved):
Good Job 👍
